### DISCUSSION TITLE
Module request - shell level

### DISCUSSION THREAD

**Discussion by SiJiL82** [JanDeDobbeleer/oh-my-posh#3078] [Ideas]
Would it be possible to have a shell level module? That shows the output of `$SHLVL` when it's > 1.
Example:
![image](https://user-images.githubusercontent.com/65114782/201658655-ff9fdbf8-c2b9-4781-8950-0810c4d26c85.png)

We have a helper script at work that creates a subshell and authenticates against a remote service in that subshell. You need to remember to exit out of the subshell when done, but if you run the script, do your tasks then come back later and can't remember if you quit out of the subshell or not, you can inadvertently close the main shell, or sit running other things inside the authenticated shell.

Having `folder_path > 2` at the end of the prompt would be fantastic.

**Comment by JanDeDobbeleer** [comment]
@SiJiL82 you can make use of the function overrides and [populate an environment variable](https://ohmyposh.dev/docs/configuration/templates#environment-variables).
